source/includes/php-mongodb-compatibility-table.rst

```diff
@@ -12,16 +13,21 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - ext 1.8 + lib 1.7
+
+
    * - ext 1.7 + lib 1.6
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
+     - |checkmark|
```

Comment:
Same comment as earlier. This row is an _existing_ release and should not report MongoDB 4.4 compatibility. Replace this checkmark with an empty first column.